docs/synthetics/faq.md

```diff
@@ -87,6 +87,52 @@ Sao Paulo (sa-east-1)      - 54.207.1.35    <br/>
 ### Where can I see if a certain device is supported for browser monitors?
 You can simply type the name of your desired device into the _Device type_ dropdown while creating a monitor and see if a matching option will be displayed. You can also find the list of devices we support, as well as their specific properties, in the official [Puppeteer docs](https://github.com/puppeteer/puppeteer/blob/v13.6.0/src/common/DeviceDescriptors.ts).
 
+### Can I extract a token from a request and use it for another request?
+Yes. Create a [Browser Monitor](https://sematext.com/docs/synthetics/browser-monitor/) and select Monitor a User Journey. Within the script you can grab a token by using the ```javascriptsetRequestInterception``` function.
+Intercept the next request, add your credentials to the request body with the following code.
+To mask your passwords and hide sensitive data please see [Storing User Journey Script Credentials Securely](https://sematext.com/docs/synthetics/user-journey-scripts/#storing-your-user-journey-script-credentials-securely)
```

Comment:
I suggest moving this since the line before ends with _"with the following code"_, so it doesn't break up the flow. I've moved it to the bottom of the example, looks better in my opinion, but it's up to you
```suggestion
```